controls/dex_works.rb

```diff
@@ -8,25 +8,25 @@
   title 'Ensure dex works as expected'
   desc '
   Verify dex by ensuring that
-  (1) its installation directory exists 
+  (1) its installation directory exists
   (2) it returns the expected version
   '
-  
+
   plan_installation_directory = command("hab pkg path #{plan_origin}/#{plan_name}")
   describe plan_installation_directory do
     its('exit_status') { should eq 0 }
     its('stdout') { should_not be_empty }
     its('stderr') { should be_empty }
   end
-  
+
   plan_pkg_version = plan_installation_directory.stdout.split("/")[5]
   ["dex"].each do |binary_name|
     command_full_path = File.join(plan_installation_directory.stdout.strip, "bin", binary_name)
     describe command("#{command_full_path} version") do
       its('exit_status') { should eq 0 }
       its('stdout') { should_not be_empty }
-      its('stdout') { should match /dex Version:\s+v#{plan_pkg_version}/ }
+      its('stdout') { should match /Dex Version:\s+v#{plan_pkg_version}\nGo Version: go1.15\nGo OS\/ARCH: linux amd64\n/}
```

Comment:
We probably just need to check the `dex` version here. Future releases may differ in go version and all we care about is that the binary executes and returns the expected version.

Reply:
Fixed. Checking only dex version